doc/extending/extending_theano.txt

```diff
@@ -880,7 +880,14 @@ Documentation
 -------------
 
 See :ref:`metadocumentation`, for some information on how to generate
-the documentation.
```

Comment:
Please change the name of the section:

```
.. _Documentation:

Documentation and coding style
------------------------------
```

The first line adds an anchor with the old name, so that if external pages were pointing to this documentation the link doesn't get broken.